pommerman/envs/v3.py

```diff
@@ -0,0 +1,110 @@
+"""The Pommerman v3 Environment, which implements Sudden Death mode.
+
+The game ends when any agent dies. In FFA, all other agents win. In any of the team
+modes, the team with the most remaining players wins. If all players die at the same
+time or the same number of players die on each time, the game ends in a tie. All
+players/teams win if the game ends without anyone dying.
```

Comment:
Shouldn't all players/teams lose if the game ends without dying? Otherwise, the best strategy is to not try.

Reply:
Up until 2 days ago, the tie reward was 0s for all players. I will change this to reflect that recent change.